.claude/skills/prototype-tickets/splitting.md

```
@@ -0,0 +1,97 @@
+- "Selecting [option] filters [table/list]"
+- "Submitting [form] creates [resource] and navigates to [page]"
+- "Deleting [resource] shows confirmation modal"
+
+**Standard ACs** — always append to Stories:
+- "Add/Update Cypress mocked tests."
+- "Add/Update unit tests for hooks/functions."
+
+## Ticket format
+
+**Stories:**
+```markdown
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)